Sources/Disposable.swift

```diff
@@ -112,7 +112,10 @@ public final class CompositeDisposable: Disposable {
 	private var state: UnsafeAtomicState<DisposableState>
 
 	/// Represents a handle to a disposable previously added to a
-	/// CompositeDisposable.
+	/// `CompositeDisposable`.
+	///
+	/// - note: Use `add(_:)` methods of `CompositeDisposable` instance to
```

Comment:
One does not intentionally create a handle though... Perhaps it should be phrased as "`add` returns a handle" instead?